website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,268 @@
+
+These permissions are found in the directory `config/rbac/`. The `ClusterRole` and `ClusterRoleBinding` manifests are responsible for setup the permissions which allow the operator to have access to the resources.
+
+**NOTE** The other files that can be found in this directory not requires changes to achieve this goal. The RBAC manifests with the names `<kind>_editor_role.yaml` and `<kind>_viewer_role.yaml` are not used by the operator and are meant to be helpers for specific config to allow access to users and clients per Kind. Also, the files with the names `auth_proxy_*.yaml` are specific to the [kube-auth-proxy][kube-rbac-proxy]. 
+
+### Changing the permissions 
+
+To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use a `Role` instead of `ClusterRole` and `RoleBinding` instead of c
+ 
+[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/cache/memcached_controller.go`)
+are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default markers don't specify a `namespace` property and will result in a ClusterRole.
+
+Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role` instead of a `ClusterRole`.
+
+Replace:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+```
+
+With the Namespace `marker`:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds/status,verbs=get;update;patch
+```
+
+And then, run `make manifests` to update `config/rbac/role.yaml`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+```
+
+<!-- todo(camilamacedo86): remove the note or improve the above info when we have further info in the issue tracked  -->     
+**NOTE**: Note that the above steps are not updating the `ClusterRoleBinding` to `RoleBinding` in `config/rbac/role_binding.yaml` which need to be done manually and shows a bug. More info: https://github.com/kubernetes-sigs/kubebuilder/issues/1496.
+
+## Using environment variables for Namespace 
+
+Instead of having the specific Namespace(s) hard-coded in the `main.go` file a good practice is to use environment 
+variables to allow the restrictive configurations
+
+### Configuring Namespace scoped operators
+
+- Add a helper function in the `main.go` file:
+
+```go
+// getWatchNamespace returns the Namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
```

Comment:
Better to recommend [`k8sutil.GetWatchNamespace()`](https://github.com/operator-framework/operator-sdk/blob/master/pkg/k8sutil/k8sutil.go#L46).

Reply:
We have not k8sutil in KB layout.

Reply:
`k8sutil` is a library that is meant to be imported. I'm not aware of this library being removed. I'd rather see a user use library code, although this function is pretty straightforward.

Reply:
We have no instructions in the quick start and in any other places telling for users add SDK module as dep lib and we do not know what we will or not port from there to KB layout. So, I do not think that we should add it here. Is it make sense? 

c/c @hasbro17 @joelanford wdyt?